packages/edit-site/src/components/list/index.js

```diff
@@ -1,25 +1,58 @@
 /**
  * WordPress dependencies
  */
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
 import { InterfaceSkeleton } from '@wordpress/interface';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { useViewportMatch } from '@wordpress/compose';
+import { store as keyboardShortcutsStore } from '@wordpress/keyboard-shortcuts';
 
 /**
  * Internal dependencies
  */
+import useRegisterShortcuts from './use-register-shortcuts';
 import Header from './header';
 import NavigationSidebar from '../navigation-sidebar';
 import Table from './table';
 
 export default function List( { templateType } ) {
+	useRegisterShortcuts();
 	const isDesktopViewport = useViewportMatch( 'medium' );
 
+	const { previousShortcut, nextShortcut } = useSelect( ( select ) => {
+		return {
+			previousShortcut: select(
+				keyboardShortcutsStore
+			).getAllShortcutKeyCombinations( 'core/edit-site/previous-region' ),
+			nextShortcut: select(
+				keyboardShortcutsStore
+			).getAllShortcutKeyCombinations( 'core/edit-site/next-region' ),
+		};
+	}, [] );
+
+	const postType = useSelect(
+		( select ) => select( coreStore ).getPostType( templateType ),
+		[ templateType ]
+	);
+
+	const itemsListLabel = postType?.labels?.items_list;
```

Comment:
I think it would be `undefined` when initially rendered, that would make the labels below having something like `"undefined - Header"` in their strings. I wonder if we should just render `null` if `postType` isn't available yet?

Reply:
I think it's preloaded. Didn't realise it could still be `null`.

Wouldn't we also have this issue elsewhere in the UI?

For the sake of this PR, I'll make it fallback to the normal labels (which are just 'Header', 'Content').

Reply:
Yeah probably. I think it should not be `null` if it's preloaded. That's a separate issue though.